website/source/docs/telemetry/index.html.md

```diff
@@ -109,6 +109,18 @@ when retrieving metrics using the above described signals.
     <td>Raft transactions / `interval`</td>
     <td>Counter</td>
   </tr>
+  <tr>
+    <td>`nomad.raft.lastIndex`</td>
+    <td>Index of the last log</td>
```

Comment:
it's the index of the last log or snapshot, I assume there are cases where the snapshot can lead the logs. Since snapshot is just a view of the logs, this may not be a meaningful distinction